tests/auth_tests/test_forms.py

```diff
@@ -439,6 +439,10 @@ class CustomAuthenticationForm(AuthenticationForm):
         username_field = User._meta.get_field(User.USERNAME_FIELD)
         self.assertEqual(form.fields['username'].label, capfirst(username_field.verbose_name))
 
+    def test_username_field_autocapitalize_none(self):
+        form = AuthenticationForm(None)
+        self.assertEqual(form.fields['username'].widget.attrs['autocapitalize'], 'none')
```

Comment:
Maybe
```python
self.assertEqual(form.fields['username'].widget.attrs.get('autocapitalize'), 'none')
```
to prevent crash when `'autocapitalize'` is not in `widget.attrs`.